tmdb/episode.go

```diff
@@ -83,15 +83,21 @@ func (episode *Episode) SetArt(show *Show, season *Season, item *xbmc.ListItem)
 		item.Art = &xbmc.ListItemArt{}
 	}
 
+	// Episode only have Still aka Thumbnail, thus we take other artworks from the season/show
+	season.SetArt(show, item)
+
+	imageQualities := GetImageQualities()
+
 	if episode.StillPath != "" {
-		item.Art.FanArt = ImageURL(episode.StillPath, "w1280")
-		item.Art.Banner = ImageURL(episode.StillPath, "w1280")
-		item.Art.Poster = ImageURL(episode.StillPath, "w1280")
-		item.Art.Thumbnail = ImageURL(episode.StillPath, "w1280")
-		item.Art.TvShowPoster = ImageURL(episode.StillPath, "w1280")
-	} else {
-		// Use the season's artwork as a fallback
-		season.SetArt(show, item)
+		item.Art.Thumbnail = ImageURL(episode.StillPath, imageQualities.Thumbnail)
+	}
+
+	//if show is very new, we might not have some artwork, thus we use Still
```

Comment:
Can we, please, make all comments having a space after `//`

Reply:
done